cirq/ops/common_gates_test.py

```diff
@@ -152,6 +153,16 @@ def test_x_matrix():
                        np.array([[1 - 1j, 1 + 1j], [1 + 1j, 1 - 1j]]) / 2)
 
 
+def test_h_matrix():
+    ket_plus = np.array([[1/np.sqrt(2), 1/np.sqrt(2)],
+                         [1/np.sqrt(2), -1/np.sqrt(2)]])
+
```

Comment:
A good one to test is that the square of (H**0.5).matrix() is nearly H.matrix().